tests/globalThis.test

```diff
@@ -0,0 +1,2 @@
+args: tests/globalThis.ts --reload
+output: tests/globalThis.ts.out
```

Comment:
Is it possible to test this in the unit tests instead? integration test are expensive to run.

Reply:
Yup. Will do.